src/abstract-view.js

```diff
@@ -6,8 +6,23 @@
 // you won't ever need to extend from this class. Use the other Views
 // to build new classes.
 //
-
-Marionette.AbstractView = Backbone.View.extend({
+import Backbone                 from 'backbone';
+import _                        from 'underscore';
+import Behaviors                from './behaviors';
+import _getValue                from './utils/_getValue';
+import getOption                from './utils/getOption';
+import normalizeMethods         from './utils/normalizeMethods';
+import normalizeUIKeys          from './utils/normalizeUIKeys';
+import normalizeUIValues        from './utils/normalizeUIValues';
+import mergeOptions             from './utils/mergeOptions';
+import proxyGetOption           from './utils/proxyGetOption';
```

Comment:
Hmm, looks like that's not yet possible with a directory. We could add an index.js file in the folder, but i don't think that would help in the long run